install_script.sh.template

```diff
@@ -1747,6 +1748,18 @@ function manage_client_libraries_security_config(){
     $sudo_cmd sed -i -n -e '/^DD_APPSEC_SCA_ENABLED=/!p' -e "\$aDD_APPSEC_SCA_ENABLED=$sca_enable" "$etc_environment"
   fi
 }
+function manage_client_libraries_profiling_config(){
+  local sudo_cmd="$1"
+  local etc_environment="$2"
+  local profiling_enable="$3"
+  if [ -n "$profiling_enable" ]; then
+    if [ ! -s "$etc_environment" ]; then
+      $sudo_cmd echo "DD_PROFILING_ENABLED=$profiling_enable" > "$etc_environment"
+    else
+      $sudo_cmd sed -i -n -e '/^DD_PROFILING_ENABLED=/!p' -e "\$aDD_PROFILING_ENABLED=$profiling_enable" "$etc_environment"
+    fi
+  fi
```

Comment:
AFAICS there is currently no way to disable profiling through the script

Reply:
It is true that there is no way to remove the variable from `/etc/environment` with the script, but invoking the script with `DD_PROFILING_ENABLED=false` will set that value in it, which will disable profiling for all processes that aren't launched with the env var overridden to a different value.

Reply:
Oh wait, I think I understand your [previous comment](https://github.com/DataDog/agent-linux-install-script/pull/220#discussion_r1631068976) now. You'd expect the absence of the env var in the installer invocation to explicitly disable profiling? I could even accept this to be the expected semantics, but then it'd work differently than the currently also supported `DD_APPSEC_ENABLED` and `DD_IAST_ENABLED` vars work. I think all three should be treated identically by the script.

Reply:
Oh indeed I completely forgot about explicitly passing `DD_PROFILING_ENABLED=false`
But as you guessed I was mostly concerned by the fact that not passing the variable will not disable profiling.

I haven't looked into the other 2 variables you mentioned, but I agree it would make sense to keep a common behavior for all 3.
I suppose this can be changed in a future PR

Reply:
The use case of removing the env var could still makes sense as a fallback mechanism.
Though I am not sure of the value versus switching from `true` to `false`.